.github/workflows/acceptance-tests.yml

```diff
@@ -98,8 +98,8 @@ jobs:
       mongodb_atlas_project_owner_id: ${{ inputs.atlas_cloud_env == 'qa' && vars.MONGODB_ATLAS_PROJECT_OWNER_ID_QA || vars.MONGODB_ATLAS_PROJECT_OWNER_ID }}
       mongodb_atlas_teams_ids: ${{ inputs.atlas_cloud_env == 'qa' && vars.MONGODB_ATLAS_TEAMS_IDS_QA || vars.MONGODB_ATLAS_TEAMS_IDS }}
       azure_atlas_app_id: ${{ inputs.atlas_cloud_env == 'qa' && vars.AZURE_ATLAS_APP_ID_QA || vars.AZURE_ATLAS_APP_ID }}
-      azure_service_principal_id: ${{ inputs.atlas_cloud_env == 'qa' && vars.AZURE_SERVICE_PRINCIPAL_ID_QA || vars.AZURE_SERVICE_PRINCIPAL_ID }}
-      azure_tenant_id: ${{ inputs.atlas_cloud_env == 'qa' && vars.AZURE_TENANT_ID_QA || vars.AZURE_TENANT_ID }}
+      azure_service_principal_id: ${{ inputs.atlas_cloud_env == 'qa' && vars.AZURE_SERVICE_PRINCIPAL_ID_APP_QA || vars.AZURE_SERVICE_PRINCIPAL_ID }}
+      azure_tenant_id: ${{ vars.AZURE_TENANT_ID }}
```

Comment:
to confirm, where you able to run all tests against QA to ensure the fix is working? Asking because the [linked execution in the PR description](https://github.com/mongodb/terraform-provider-mongodbatlas/actions/runs/11391433474/job/31695292147) ran against dev (see MONGODB_ATLAS_BASE_URL value in setup step)

Reply:
You are right 😱
Re-running here: https://github.com/mongodb/terraform-provider-mongodbatlas/actions/runs/11400250945/job/31720710293

Reply:
Output from the run ✅ 
Tests using the `AZURE_TENANT_ID`
  - TestAccBackupSnapshotExportBucket_basicAzure, [OK](https://github.com/mongodb/terraform-provider-mongodbatlas/actions/runs/11400250945/job/31720710691#step:5:141)
  - TestAccCloudProviderAccessAuthorizationAzure_basic, [OK](https://github.com/mongodb/terraform-provider-mongodbatlas/actions/runs/11400250945/job/31720711633#step:5:250)
  - TestAccCloudProviderAccessSetupAzure_basic, [OK](https://github.com/mongodb/terraform-provider-mongodbatlas/actions/runs/11400250945/job/31720711633#step:5:261)
  - TestAccEncryptionAtRest_azure_requirePrivateNetworking, [OK](https://github.com/mongodb/terraform-provider-mongodbatlas/actions/runs/11400250945/job/31720712172#step:5:87)
  - TestAccEncryptionAtRest_basicAzure, [OK](https://github.com/mongodb/terraform-provider-mongodbatlas/actions/runs/11400250945/job/31720712172#step:5:67)
  - TestAccEncryptionAtRestPrivateEndpoint_approveEndpointWithAzureProvider, 
    - `SKIPPED`, also skipped in [dev](https://github.com/mongodb/terraform-provider-mongodbatlas/actions/runs/11375947327/job/31663953551#step:5:82)
  - TestAccEncryptionAtRestPrivateEndpoint_basic, [OK](https://github.com/mongodb/terraform-provider-mongodbatlas/actions/runs/11400250945/job/31720712172#step:5:80)
  - TestAccEncryptionAtRestPrivateEndpoint_transitionPublicToPrivateNetwork, [OK](https://github.com/mongodb/terraform-provider-mongodbatlas/actions/runs/11400250945/job/31720712172#step:5:85)
  - TestMigEncryptionAtRest_basicAzure, [OK](https://github.com/mongodb/terraform-provider-mongodbatlas/actions/runs/11400250945/job/31720712172#step:5:56)


Tests using the `AZURE_SERVICE_PRINCIPAL_ID`
  - TestAccBackupSnapshotExportBucket_basicAzure (covered above)
  - TestAccCloudProviderAccessAuthorizationAzure_basic (covered above)
  - TestAccCloudProviderAccessSetupAzure_basic (covered above)

Reply:
great, thanks for double checking

Reply:
Ran skipped test locally:
```
--- PASS: TestAccEncryptionAtRestPrivateEndpoint_approveEndpointWithAzureProvider (525.40s)
```